src/main/sources/VcfDataSource.js

```diff
@@ -91,13 +92,16 @@ function createFromVcfFile(remoteSource: VcfFile): VcfDataSource {
   return o;
 }
 
-function create(data: {url:string}): VcfDataSource {
+function create(data: Object): VcfDataSource {
```

Comment:
sure,

As I mentioned. I'm not JavaScript developer and Ecmascript 2015 is definitely new to me. I didn't know that I can do that.